telemetry/definitions/commonDefinitions.json

```diff
@@ -122,6 +122,11 @@
             "name": "aws_createCredentials",
             "description": "Create a new credentials file"
         },
+        {
+            "name": "aws_injectCredentials",
+            "description": "Inject selected AWS credentials into a third-party run (e.g. RunConfiguration)",
+            "metadata": [{ "type": "result" }]
```

Comment:
should we care about flavor? python vs java vs node? As an optional field?

Reply:
I was wondering that too ... good call

Reply:
so runtime?

Reply:
sounds fine to be cocsistent

Reply:
yeah - no - I've added a `runtimeString` value that is free text so that we don't have to introduce new logic when a new version of Node comes out or whatever.